core/templates/services/UpgradedServices.ts

```diff
@@ -33,6 +33,10 @@ import { AdminRouterService } from
 import { AdminTaskManagerService } from
   'pages/admin-page/services/admin-task-manager.service';
 import { AlertsService } from 'services/alerts.service';
+import { AdminBackendApiService } from
```

Comment:
Please add these imports in sorted manner as it is done in this file already.

Reply:
done